test/controllers/content_items_controller_test.rb

```diff
@@ -130,6 +130,34 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_equal content_item['title'], assigns[:content_item].title
   end
 
+  test "gets item from content store and keeps existing ordered_related_items when feature flag header not specified" do
+    content_item = content_store_has_schema_example('case_study', 'case_study')
+
+    get :show, params: { path: path_for(content_item) }
+    assert_response :success
+    assert_equal content_item['links']['ordered_related_items'], assigns[:content_item].content_item['links']['ordered_related_items']
+  end
+
+  test "gets item from content store and keep existing ordered_related_items when feature flag header is specified but links already exist" do
+    request.headers[FeatureFlagNames.recommended_related_links] = 'true'
+
+    content_item = content_store_has_schema_example('guide', 'guide')
+
+    get :show, params: { path: path_for(content_item) }
+    assert_response :success
+    assert_equal content_item['links']['ordered_related_items'], assigns[:content_item].content_item['links']['ordered_related_items']
+  end
+
+  test "gets item from content store and replaces ordered_related_items when feature flag header is specified and there are no existing links" do
+    request.headers[FeatureFlagNames.recommended_related_links] = 'true'
+
+    content_item = content_store_has_schema_example('case_study', 'case_study')
+
+    get :show, params: { path: path_for(content_item) }
+    assert_response :success
+    assert_equal assigns[:content_item].content_item['links']['ordered_related_items'], content_item['links']['suggested_ordered_related_items']
+  end
```

Comment:
I'm not sure why this test and the one above it are testing different things.  Shouldn't you explicitly set the `ordered_related_items` in the above test and delete it in this one?

If `guide` and `case_study` just happen to have the property that examples of the former have `ordered_related_items` and examples of the latter don't, I still think it should be made explicit here.

Reply:
I see where you're coming from - they are testing different things because one of the content schema examples has existing `ordered_related_items` and the other has no `ordered_related_items` but does have `suggested_ordered_related_items`. This was through an intentional change to the content schemas some time ago, and it is necessary to test these behaviours as we don't want to be overriding existing related links where they exist.

To your second point, how do you feel this would be best made explicit?

Reply:
Maybe something like

```
# first test
content_item = ...
content_item['links']['ordered_related_items'] = [...]
content_item['links']['suggested_ordered_related_items'] = [...]
# check the ordered_related_items is unchanged

# first test
content_item = ...
content_item['links']['ordered_related_items'] = []
content_item['links']['suggested_ordered_related_items'] = [...]
# check the ordered_related_items is changed
```

I don't see why overwriting the links would be a problem here.